scripts/yg-privacyenhanced.py

```diff
@@ -66,6 +67,9 @@ def create_my_orders(self):
                                              self.txfee * (1 + float(self.txfee_factor))))
         randomize_minsize = int(random.uniform(self.minsize * (1 - float(self.size_factor)),
                                                self.minsize * (1 + float(self.size_factor))))
+        if randomize_minsize < jm_single().DUST_THRESHOLD:
+            jlog.warn("Minsize was randomized to below dust; resetting to dust threshold.")
```

Comment:
Why "warning"? Could be "info" IMO.

Also should output `DUST_THRESHOLD` value too.

Reply:
I think warning makes sense because the user is making a "bad" decision, and we are correcting it for them.
For the second point, yeah, I will add that now.